tests/yam/validate/validate_installed_disk.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that the product installed at expected disk
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use scheduler 'get_test_suite_data';
+
+sub run {
+    select_console 'root-console';
+
+    my $installed_disk_expected = get_test_suite_data()->{installed_disk};
```

Comment:
I think validate vda is not useful as its the default, so we don't know if the disk was selected thanks to the jsonnet code, or just because it's the default.
Please use a second disk, then you could try to use console/validate_first_disk_selection with this test data: `
  guided_partitioning:
    disks:
      - vdb
  unused_disks:
    - vda`

Reply:
It already has 'console/validate_first_disk_selection' test module to check the disk.
I will delete this test module, use the existed one.